Sponsors.md

```diff
@@ -1,90 +1,91 @@
 
-## DevStaff Swag Sponsor
-Help the community members wear the DevStaff community name with pride! With this option, you will sponsor the purchase of promotional material with the DevStaff brand, in return for awareness of your own brand.  (Please note we do not currently offer a co-branding option, see below for awareness benefits). All material will be made available to community and org team members.
+As a thank you for sponsoring a number of next meetups, we will feature you in those meetups by mentioning you and your contribution.
 
-The swag pack will be awarded to lucky attendees through a lottery that will take place at the end of the meetup. Your company will be publicly thanked for making this possible. 
+### How financial contributions are used
 
-The swag pack will include: 
-* a dev-friendly cool looking t-shirt, 
-* “Raki” / “Tsikoudia” shot glasses! (we are the definitive developer community in Crete after all) 
-* (optionally) any other item that can fit in the budget
+The financial contributions may be used by DevStaff for various purposes to support the community, e.g., providing food at a meetup, ordering swag to give out to community members, etc. 
+These activities are expensed in accordance to OpenCollective's policies, including submitting a receipt for transparency purposes.
 
-Here’s how the swag draw will work: 
-* 5 swag packs will be awarded at each meetup
-* Each pack costs 20 EUR
-* 100 EUR buys us 5 packs - i.e. awareness for you for ONE meetup. 
-* 200 EUR buys us 10 packs - i.e. awareness for you for TWO meetups. 
-* 400 EUR buys us 20 packs - i.e. awareness for you for FOUR meetups. 
+## Other ways to sponsor
+
+We provide a few other ways to sponsor DevStaff outside OpenCollective, that provides the advantage to a legal entity (e.g., a company) to be directly invoiced. Your contribution will be made aware to our community (e.g., during meetups), but note that it cannot be listed in our OpenCollective page.
+
+### Swag Sponsor
+
+Help the community members wear the DevStaff community name with pride! With this option, you can sponsor the purchase of promotional material with the DevStaff brand, in return for awareness of your own brand.  (Please note we do not currently offer a co-branding option, see below for awareness benefits). All material will be made available to community and org team members.
+
+Each swag pack may include a dev-friendly cool looking t-shirt, and/or "Raki" / "Tsikoudia" shot glasses, and/or any other item that can fit in the budget.
+
+Each swag pack will be awarded to lucky attendees through a lottery that will take place at the meetup. Your company will be publicly thanked for making this possible. 
+
+Get in contact with us to discuss swag pack opportunities. Please note that depending on the amount of swag packs, we will publicly thank you in a number of next meetups. For example:
+
+* 5 swag packs - i.e. awareness for you for ONE meetup. 
```

Comment:
Maybe we could keep this more generic by using the amount of money spEnt/cost of the SWAG.
E.g. 100 € per meetup (leaflets, thank you message at meetup and social media).

Reply:
Tried EUR amounts instead. Please review